### PR TITLE
Extract AuthTokenInput for reuse on any form

### PR DESCRIPTION
**Pull request by exbinary**
Tiny PR extracting an `AuthTokenInput` component.

### CODE REVIEW

File: app/javascript/components/NavBar.vue [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 <script>
 import logo from 'images/logo.png'
-import DeleteButton from 'components/DeleteButton'
+import DeleteButton from './DeleteButton'
```
Comment (maebeale) [inline app/javascript/components/NavBar.vue:33]: is this from another PR?

Comment (maebeale) [inline app/javascript/components/NavBar.vue:33]: Just curious -- why is this needed/available now?

Comment (exbinary) [reply app/javascript/components/NavBar.vue:33]: It was not part of the other changes here. I just noticed that `NavBar` and `DeleteButton` are siblings, so this is simpler and helps IDEs 'go-to' the file (they often can't figure out where `components/` is).
